day2.md

```diff
@@ -0,0 +1,171 @@
+
+After enough wondering and calculation, we picked<sup>[2](#pick)</sup> a small sushi shop and ate there.
+<img src="day2/IMG_20160906_110423.jpg" width="500"><br/>
+The "restaurant" was very small. We sat on the other side of it, but that's pretty much it. Most "restaurants" in the midst of the market were about the same size. For some reason, my brain just connect these type of shops with adjectives such as "authentic" or "local"<sup>[3](#small)</sup>.
+
+<img src="day2/IMG_20160906_104720.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_105812.jpg" width="500"><br/>
+Here comes our sashimi bowel and salmon sashimi!<br/>
+They were good but a bit pricy and didn't live up to our expectations.
+
+<img src="day2/IMG_20160906_111430.jpg" width="500"><br/>
+We also grabbed some of these before leaving the market!😄<br/>
+One of my favorite snacks in Japan, I miss them!
+
+We then headed to  <a name="skytree">[Tokyo Skytree Town](http://www.tokyo-solamachi.jp/english/floor/)</a>!
+<br/><img src="day2/IMG_20160906_120835.jpg" width="500"><br/>
+<br/><img src="day2/IMG_20160906_121403.jpg" width="500"><br/>
+First thing we saw after we got out from the station was a store for this famous banana bread.
+
+The bottom floors of the skytree tower includes various areas such as a shopping district, a fashion zone, and a food court.
+
+We randomly walked into a manga/anime shop packed with manga and related products! <br />
+Grown up watch anime, the place recalled a lot of good old time for me.
+
+<img src="day2/IMG_20160906_124543.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_124559.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_124714.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_124721.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_124747.jpg" width="500"><br/>
+So many One Piece stuff!!! 😱😱😱<br/>
+
+<img src="day2/IMG_20160906_125154.jpg" width="500"><br/>
+<img src="day2/14435302_1115753031807723_6380156596113072249_o.jpg" width="500"><br/>
+They labeled the actual heights of some famous manga characters.<br />
+I'm pretty sure 越前リョーマ from The Prince of Tennis, the anime that got me into playing tennis, grown over 151cm after middle school.😓<br />
+Matt is about the same heigh as Luffy! lol 😂
+
+<img src="day2/IMG_20160906_132543.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_132619.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_132948.jpg" width="500"><br/>
+One of my favorite anime: Ojarumaru (反斗小王子)! I watched it when I was in preschool living with my aunt and cousin! 😄 I even watched a bit of it on my phone on Youtube when we got back to our airbnb place that day just to recall some good old memories. 😂
+
+<img src="day2/14380199_1115753295141030_5126574712730509424_o.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_133019.jpg" width="500"><br/>
+Don't know what's his name, but I saw him everwhere and he is so cute!
```

Comment:
- "what his name is" is more natural
- "everywhere"

(Also, it's Domo. hahaha)